command/services/named_ip.go

```diff
@@ -0,0 +1,143 @@
+package services
+
+import (
+	"github.com/urfave/cli"
+	"kube-helper/service"
+	"google.golang.org/api/compute/v1"
+	"strings"
+)
+
+func CmdAddRuleToLoadBalancer(c *cli.Context) error {
+	configContainer, err := configLoader.LoadConfigFromPath(c.String("config"))
+
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	projectId = configContainer.Cluster.ProjectID
+	if projectId == "" {
+		return cli.NewExitError("projectId is empty", 1)
```

Comment:
if this is required than it is better to put in der config loader instead of outside checks